jetstream_pt/engine.py

```diff
@@ -630,7 +626,7 @@ def samples_per_slot(self) -> int:
 
   @property
   def max_prefill_length(self) -> int:
-    return self.param.max_seq_len
+    return self.env.max_input_sequence_length
```

Comment:
Great, it's much clear now. Engineer might need to figure out and do mapping max_seq_len is max_prefill_length